validate_sptx/util.py

```diff
@@ -107,16 +114,202 @@ def validate_object(self, target_object: Dict, target_file: str=None) -> bool:
             loaded json object to be validated against the schem passed to this object's constructor
         target_file : str
             informational string regarding the source file of the given object
+        fuzz : bool
+            whether or not to perform element-by-element fuzzing.
+            If true, will return true and will *not* use warnings.
 
         Returns
         -------
         bool :
-            True, if object valid, else False
+            True, if object valid or fuzz=True, else False
 
         """
+
+        if fuzz:
+            if target_file:
+                print(f"> Fuzzing {target_file}...")
+            else:
+                print("> Fuzzing unknown...")
+            fuzzer = Fuzzer(self._validator, target_object)
+            fuzzer.fuzz()
+            return True
+
         if self._validator.is_valid(target_object):
             return True
         else:
             es: Iterator[ValidationError] = self._validator.iter_errors(target_object)
             self._recurse_through_errors(es, filename=target_file)
             return False
+
+
+class Fuzzer(object):
+
+    def __init__(self, validator : Draft4Validator, obj : Any, out : IO=sys.stdout) -> None:
```

Comment:
No space before colon, e.g.: 
```diff    
- def __init__(self, validator : Draft4Validator, obj : Any, out : IO=sys.stdout) -> None:
+ def __init__(self, validator: Draft4Validator, obj: Any, out: IO=sys.stdout) -> None:
```

Reply:
Sorry, I was inadequately specific here -- this is the convention for the `__init__` block, the _docstrings_ should have spaces.

... that convention is confusing. Sorry. :(